src/Microsoft.Extensions.Logging.Analyzers/LogFormatAnalyzer.cs

```diff
@@ -113,7 +120,10 @@ private void AnalyzeFormatArgument(SyntaxNodeAnalysisContext syntaxContext, Expr
                 if (int.TryParse(valueName, out _))
                 {
                     syntaxContext.ReportDiagnostic(Diagnostic.Create(Descriptors.MEL1NumericsInFormatString, formatExpression.GetLocation()));
-                    break;
```

Comment:
I couldn't see a reason for this `break`, it would mean you only get one diagnostic per format string, but I'd rather emit all of the diagnostics to avoid new errors appearing when you fix one.